hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/metadata/TestHoodieBackedMetadata.java

```diff
@@ -444,7 +445,8 @@ public void testSync(HoodieTableType tableType) throws Exception {
       assertTrue(metadata(client).isInSync());
     }
 
-    // Various table operations without metadata table enabled
+    // Various table operations without metadata table enabled. When metadata is disabled, file system
```

Comment:
but these calls seem like they enable metadata? first arg of `getWriteConfig(..)`

Reply:
The second arg is useFilelistingMetadata = false